grafanalib/opentsdb.py

```diff
@@ -0,0 +1,156 @@
+import attr
+from attr.validators import instance_of
+
+# OpenTSDB aggregators
+OTSDB_AGG_AVG = "avg"
+OTSDB_AGG_COUNT = "count"
+OTSDB_AGG_DEV = "dev"
+OTSDB_AGG_EP50R3 = "ep50r3"
+OTSDB_AGG_EP50R7 = "ep50r7"
+OTSDB_AGG_EP75R3 = "ep75r3"
+OTSDB_AGG_EP75R7 = "ep75r7"
+OTSDB_AGG_EP90R3 = "ep90r3"
+OTSDB_AGG_EP90R7 = "ep90r7"
+OTSDB_AGG_EP95R3 = "ep95r3"
+OTSDB_AGG_EP95R7 = "ep95r7"
+OTSDB_AGG_EP99R3 = "ep99r3"
+OTSDB_AGG_EP99R7 = "ep99r7"
+OTSDB_AGG_EP999R3 = "ep999r3"
+OTSDB_AGG_EP999R7 = "ep999r7"
+OTSDB_AGG_FIRST = "first"
+OTSDB_AGG_LAST = "last"
+OTSDB_AGG_MIMMIN = "mimmin"
+OTSDB_AGG_MIMMAX = "mimmax"
+OTSDB_AGG_MIN = "min"
+OTSDB_AGG_MAX = "max"
+OTSDB_AGG_NONE = "none"
+OTSDB_AGG_P50 = "p50"
+OTSDB_AGG_P75 = "p75"
+OTSDB_AGG_P90 = "p90"
+OTSDB_AGG_P95 = "p95"
+OTSDB_AGG_P99 = "p99"
+OTSDB_AGG_P999 = "p999"
+OTSDB_AGG_SUM = "sum"
+OTSDB_AGG_ZIMSUM = "zimsum"
+
+OTSDB_DOWNSAMPLING_FILL_POLICIES = ('none', 'nan', 'null', 'zero')
+OTSDB_DOWNSAMPLING_FILL_POLICY_DEFAULT = 'none'
+
+OTSDB_QUERY_FILTERS = (
+    'literal_or', 'ilteral_or', 'not_literal_or',
```

Comment:
Actually, I'm not sure http://opentsdb.net/docs/build/html/user_guide/query/filters.html#ilteral-or, but in grafana web UI it's `iliteral_or`.
I'll checkout opentsdb sources...